src/dbnode/storage/bootstrap/bootstrapper/uninitialized/types.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package uninitialized
+
+import (
+	"github.com/m3db/m3/src/dbnode/storage/bootstrap/result"
+	"github.com/m3db/m3x/instrument"
+)
+
+// Options is the options interface for the uninitialized source.
+type Options interface {
+	// SetResultOptions sets the result options
+	SetResultOptions(value result.Options) Options
+
+	// ResultOptions returns the result options
+	ResultOptions() result.Options
+
+	// Return the instrument options.
+	InstrumentOptions() instrument.Options
+
+	// Set the instrument options.
+	SetInstrumentOptions(value instrument.Options) Options
```

Comment:
nit: follow ordering of setter before getter for each type.

Reply:
will fix